src/main/scala/cognite/spark/v1/TimeSeriesRelation.scala

```diff
@@ -47,13 +47,12 @@ class TimeSeriesRelation(config: RelationConfig, useLegacyName: Boolean)(val sql
 
   override def getFromRowAndCreate(rows: Seq[Row]): IO[Unit] = {
     val timeSeriesSeq = rows.map { r =>
-      val timeSeries = fromRow[TimeSeries](r)
-      val timeSeriesWithMetadata = timeSeries.copy(metadata = filterMetadata(timeSeries.metadata))
-      val timeSeriesCreate = timeSeriesWithMetadata.transformInto[TimeSeriesCreate]
+      val timeSeries = fromRow[TimeSeriesCreate](r)
+      timeSeries.copy(metadata = filterMetadata(timeSeries.metadata))
```

Comment:
`.copy` does not modify the item in place, it will return a new `TimeSeriesCreate` item with the modified metadata. You need to keep `timeSeriesWithMetadata` as it was.

We should have a test for this if we didn't catch it in one of the existing ones.

Reply:
Test was correctly failing with this change, I just missed it. I changed it back to the way it was and the test is passing